package.json

```diff
@@ -29,27 +29,15 @@
         "linting"
     ],
     "activationEvents": [
-        "onLanguage:yaml"
+        "onLanguage:yaml",
+        "onLanguage:github-actions-workflow"
     ],
     "qna": "https://github.com/orgs/zizmorcore/discussions",
     "sponsor": {
         "url": "https://github.com/sponsors/woodruffw"
     },
     "main": "./dist/extension.js",
     "contributes": {
-        "languages": [
-            {
-                "id": "yaml",
-                "extensions": [
-                    ".yml",
-                    ".yaml"
-                ],
-                "aliases": [
-                    "YAML",
-                    "yaml"
-                ]
-            }
-        ],
```

Comment:
Making sure I understand: we should remove this because it clobbers other extensions, or because it's superfluous, or something else?

(This is potentially a dumb question; I'm not super familiar with VS Code extensions, and this one was a learning experience.)

Reply:
The extension isn't handling a new language, so this is a) misleading to people installing it and b) dead code.

It doesn't _break_ anything, but it's not doing anything.